test/datapipes/test_ahmed_body.py

```diff
@@ -12,23 +12,33 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import os
 import pytest
 import torch
+import importlib
 
 from typing import Tuple
-from modulus.datapipes.gnn.ahmed_body_dataset import AhmedBodyDataset
 from . import common
+from pytest_utils import import_or_fail
 
 Tensor = torch.Tensor
 
 
 @pytest.fixture
 def data_dir():
-    return "/data/nfs/modulus-data/datasets/ahmed_body/"
+    path = "/data/nfs/modulus-data/datasets/ahmed_body/"
+    if not os.path.exists(path):
```

Comment:
I think this should be a completely separate skip util like the import one...

namely:
```
def _nfsdata_or_fail(config):
       if not os.path.exists('/data/nfs/modulus-data'):
              pytest.skip(
                  "NFS volumes not set up with CI data repo. Run `make get-data` from the root directory of the repo"
              )
```

This only checks to see if the modulus-data repo exists not individual folders, this means this will error if the system is on an older commit without a specific folder. Vs the current implementation would skip

Reply:
Thanks! Added this